prowler/providers/aws/lib/s3/s3.py

```diff
@@ -147,3 +150,38 @@ def send_to_bucket(
                 f"{error.__class__.__name__}[{error.__traceback__.tb_lineno}] -- {error}"
             )
         return uploaded_objects
+
+    @staticmethod
+    def test_connection(
+        session, bucket_name: str, raise_on_exception: bool = True
+    ) -> Connection:
+        """
+        Test the connection to the S3 bucket.
+
+        Parameters:
+        - session: An instance of the `Session` class representing the AWS session.
+        - bucket_name: A string representing the name of the S3 bucket.
```

Comment:
Are we supporting also the bucket URI here?

Reply:
Good point! Done